### PR TITLE
Added PHPREdis Extension and Added Instrunction in README.md

### PR DESCRIPTION
**Pull request by g0ld3lux**

### CODE REVIEW

**Comment by g0ld3lux** [comment]
@philtrep  please merge this one for phpredis, ive fix some issue in syntax here in dockerfile70 plus added instructions in README.md and Added phpredis to be enable by default in docker-compose.yml

**Comment by appleboy** [comment]
@g0ld3lux Please fix some conflicts.

File: README.md [hunk 1]
```diff
@@ -714,8 +714,27 @@ The PHP-CLI is installed in the Workspace container. To change the PHP-CLI versi
 
 Right now you have to manually edit the `Dockerfile` or create a new one like it's done for the PHP-FPM. (consider contributing).
 
+<br>
+<a name="Install PHP Redis Extension"></a>
+### Install PHP Redis on Php 7
```
Comment (appleboy) [inline README.md]: Please also update Top Contents.

File: README.md [hunk 2]
```diff
@@ -714,8 +714,27 @@ The PHP-CLI is installed in the Workspace container. To change the PHP-CLI versi
...
+1 - Open 'docker-compose.yml' file
+<br>
```
Comment (appleboy) [inline README.md]: Please remove `<br>` and just use empty line.

File: README.md [hunk 3]
```diff
@@ -714,8 +714,27 @@ The PHP-CLI is installed in the Workspace container. To change the PHP-CLI versi
...
+php-fpm:
+    build:
+        context: ./php-fpm
+        args:
+            - INSTALL_PHPREDIS=true
 
-
+```
+3 - Re-Build the Container `docker-compose build php-fpm`
+4 - Check if PhpRedis Extension is installed properly , If You See OK then You Have Installed it Successfully!
```
Comment (appleboy) [inline README.md]: Missing empty line.

File: README.md (hunk 1 above)
Comment (appleboy) [inline README.md]: and maybe change the title to `Install PHP Redis Extension`?

**Comment by philtrep** [comment]
@g0ld3lux This PR is just missing a few changes according to @appleboy 's review.

**Comment by appleboy** [comment]
@g0ld3lux any updates on this?

**Comment by g0ld3lux** [comment]
What are the issue your are facing? anyway? let me know thanks , so i can update this

**Pull request closed without merging**